roles/ceph-osd/tasks/osd_fragment.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: get osd path
-  shell: "df | grep {{ item }} | awk '{print $6}'"
+  shell: "lsblk -no MOUNTPOINT {{ item }} | grep 'ceph-'"
```

Comment:
@czunker why using `| grep 'ceph-'` while you know the list of devices (`with_items: "{{ devices }}"`) ?

Reply:
@guits With encrypted devices, lsblk returns two entries on my systems:
`
# lsblk -no MOUNTPOINT /dev/sdb
/var/lib/ceph/osd/ceph-6
/var/lib/ceph/osd-lockbox/40727135-93d3-4002-875b-84899a7e32ea
`
I tried to include the partlabel which you used in the blkid example, but lsblk does not show the ceph-data label. So I had to limit the output by grep.

Reply:
@czunker add `--nodeps` option and you will have only 1 entry

Reply:
@guits It's not working:
```
# lsblk -no MOUNTPOINT --nodeps /dev/sdb

```
This is the complete output for one disk:
```
sdb                                        8:16   0   3.7T  0 disk  
├─sdb1                                     8:17   0   3.7T  0 part  
│ └─40727135-93d3-4002-875b-84899a7e32ea 252:1    0   3.7T  0 crypt /var/lib/ceph/osd/ceph-6
└─sdb5                                     8:21   0    10M  0 part  /var/lib/ceph/osd-lockbox/40727135-93d3-4002-875b-84899a7e32ea
```
I found a way to show the `ceph data` label, but that's assigned to the unencrypted partition:
```
sdb                                        3.7T  0 disk               
├─sdb1                                     3.7T  0 part  ceph data    
│ └─40727135-93d3-4002-875b-84899a7e32ea   3.7T  0 crypt              /var/lib/ceph/osd/ceph-6
└─sdb5                                      10M  0 part  ceph lockbox /var/lib/ceph/osd-lockbox/40727135-93d3-4002-875b-84899a7e32ea
```

Reply:
@guits @czunker   `--nodeps` gives just one entry , but only if I address the correct partition..., but I guess thats not what is in `{{ devices }}` right?
```

## Without nodeps
# lsblk -no MOUNTPOINT /dev/sdb


/var/lib/ceph/osd/ceph-9
/var/lib/ceph/osd-lockbox/ab71d7dc-2b98-4ef4-90cb-23204d7eee50
# lsblk -no MOUNTPOINT /dev/sdb1

/var/lib/ceph/osd/ceph-9
# lsblk -no MOUNTPOINT /dev/sdb5
/var/lib/ceph/osd-lockbox/ab71d7dc-2b98-4ef4-90cb-23204d7eee50




## With nodeps
# lsblk -no MOUNTPOINT --nodeps /dev/sdb

# lsblk -no MOUNTPOINT --nodeps /dev/sdb1

# lsblk -no MOUNTPOINT --nodeps /dev/sdb5
/var/lib/ceph/osd-lockbox/ab71d7dc-2b98-4ef4-90cb-23204d7eee50
```

Reply:
@D3N14L Yes, devices only includes /dev/sdb and alike.